### PR TITLE
[4083] bullet list indent to right missing fix

### PR DESCRIPTION
**Pull request by Mohamed-halim-sw**
**Related issue(s):**
* Fixes scandipwa/scandipwa/issues/4083

**Problem:**
* the bullets were not indented in relation to the text

**In this PR:**
* made style changes to "shortDescription" class in "ProuctActions.style.scss" component style
* fixed position to relative
* added margin-inline-end
* added inset-block-start

### CODE REVIEW

**Comment by aleksandrakorolova** [comment]
Fixed for short description on PDP

Still, can reproduce for:
1.  Description on PDP
Screenshot 2022-01-19 at 12 44 52
2. CMS Page
Screenshot 2022-01-20 at 12 41 23
3. Category description on PLP
Screenshot 2022-01-20 at 12 46 21

**Comment by Elena-Evsa** [comment]
Please, recreate link
image

File: packages/scandipwa/src/component/CategoryDetails/CategoryDetails.style.scss [hunk 1]
```diff
@@ -48,7 +48,15 @@
         justify-content: center;
         font-size: 14px;
         font-weight: 400;
-
+        
+
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/CategoryDetails/CategoryDetails.style.scss:52]: remove duplicate white space

File: packages/scandipwa/src/component/ProductInformation/ProductInformation.style.scss [hunk 2]
```diff
@@ -66,13 +66,24 @@
...
+            margin-inline-start: 10px;
+            margin-inline-end: -25px;
+            inset-block-start: -1px;
+        }
+
+        @include default-list;
+
+        @include mobile {
+            line-height: 20px;
+
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/ProductInformation/ProductInformation.style.scss:86]: remove this white space

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]
See comments

**Comment by Elena-Evsa** [comment]
1. Fixed
2. Fixed
3. Fixed
4. Wrong indent betwen lines with bullets on PDP now it 20 px, but should be 6px
image

**Comment by Elena-Evsa** [comment]
4. Please, Make the styling of indents as on master
Now margin from the top and botom present
Indent after paragraphs = 6 px, but should be 14 px
image
Here the example of product description of which you can compare
https://master.d25kd40decjpue.amplifyapp.com/default/bundl-with-out-of-stock-radio-buttons.html
https://master.d25kd40decjpue.amplifyapp.com/default/n-cocktail-dress.html

**Comment by Elena-Evsa** [comment]
1. Fixed
2. Fixed
3. Fixed
4. Fixed
5. Please, make margin after last bullet (now = 20px) same as after any other paragraph = 14px
image

**Comment by Elena-Evsa** [comment]
1. Fixed
2. Fixed
3. Fixed
4. Fixed
5. Fixed

File: packages/scandipwa/src/component/ProductInformation/ProductInformation.style.scss [hunk 3]
```diff
@@ -66,13 +66,21 @@
...
+            font-size: 14px;
             line-height: 20px;
         }
 
-        li {
-            &::before {
-                position: static;
-            }
+        li::before {
+            position: relative;
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/ProductInformation/ProductInformation.style.scss:74]: Make nesting as it was before
```
li {
   &::before {
   ....
   }
}
```

File: packages/scandipwa/src/component/CategoryDetails/CategoryDetails.style.scss [hunk 4]
```diff
@@ -49,6 +49,13 @@
         font-size: 14px;
         font-weight: 400;
 
+        li::before {
+            position: relative;
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/CategoryDetails/CategoryDetails.style.scss:53]: Same here

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]

File: packages/scandipwa/src/component/Html/Html.component.js [hunk 5]
```diff
@@ -250,7 +250,10 @@ export class Html extends PureComponent {
             script.appendChild(document.createTextNode(children[0].data));
         }
 
-        document.head.appendChild(script);
+        if (!Number.isNaN(Number(script))) {
+            document.head.appendChild(script);
+        }
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/Html/Html.component.js:255]: I guess it is applied in another PR

File: packages/scandipwa/src/component/ProductInformation/ProductInformation.style.scss [hunk 6]
```diff
@@ -66,13 +66,24 @@
...
         li {
             &::before {
-                position: static;
+                position: relative;
+                margin-inline-start: 10px;
+                margin-inline-end: -25px;
+                inset-block-start: -1px;
             }
         }
+
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/ProductInformation/ProductInformation.style.scss:81]: Remove this space

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]

**Review by AleksandrsKondratjevs — APPROVED** [review]

**Pull request merged**